### DISCUSSION TITLE
After adding the content in service file for saving the content I am given command :wq I am getting error

### DISCUSSION THREAD

**Discussion by Kurra432** [learndevopsonline/learndevopsonline#1517] [Q&A]
![cart 2023-10-11 114430](https://github.com/learndevopsonline/learndevopsonline/assets/129031517/29d1509a-3b26-418f-92e2-5d9bf9ccf506)

**Comment by Kurra432** [comment]
I am not able save the content in service file

**Reply by sreedharm07** [reply]
please start your machine 
its inactive in a middle while  checking

**Reply by Kurra432** [reply]
ok
